sdk/Metrics/Exporters/PrometheusExporter.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Metrics\Exporters;
+
+use OpenTelemetry\Metrics as API;
+use OpenTelemetry\Sdk\Metrics\Counter;
+use OpenTelemetry\Sdk\Metrics\Exceptions\CantBeExported;
+use Prometheus\CollectorRegistry;
+
+class PrometheusExporter extends AbstractExporter
+{
+    /**
+     * @var CollectorRegistry $registry
+     */
+    protected $registry;
+
+    /**
+     * @var string $namespace
+     */
+    protected $namespace;
+
+    public function __construct(CollectorRegistry $registry, string $namespace = '')
+    {
+        $this->registry = $registry;
+        $this->namespace = $namespace;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function doExport(iterable $metrics): void
+    {
+        foreach ($metrics as $metric) {
```

Comment:
I think we'll want to keep an eye on this nested switch as the metrics groupings get bigger.

Reply:
Discussed via Zoom